tests/src/test/java/alluxio/master/FileSystemMasterClientIntegrationTest.java

```diff
@@ -62,4 +66,26 @@ public void getFileInfoReturnsOnError() throws IOException, AlluxioException {
     fsMasterClient.getStatus(new AlluxioURI("/doesNotExist"));
     fsMasterClient.close();
   }
+
+  @Test(timeout = 300000)
```

Comment:
Is it possible to make this test expect to finish in a more reasonable amount of time?